package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "sap-cloud-sdk",
+  "version": "1.34.0",
```

Comment:
Is this mandatory for the typedoc, since lerna is not used?

Reply:
Yes.